<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
     <class>BlockOnce</class>
     <widget class="QMainWindow" name="BlockOnce">
          <property name="enabled">
               <bool>true</bool>
          </property>
          <property name="geometry">
               <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>415</width>
                    <height>287</height>
               </rect>
          </property>
          <property name="sizePolicy">
               <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
               </sizepolicy>
          </property>
          <property name="minimumSize">
               <size>
                    <width>415</width>
                    <height>287</height>
               </size>
          </property>
          <property name="maximumSize">
               <size>
                    <width>415</width>
                    <height>287</height>
               </size>
          </property>
          <property name="windowTitle">
               <string>BlockOnce</string>
          </property>
          <widget class="QWidget" name="centralwidget">
               <widget class="QWidget" name="background" native="true">
                    <property name="geometry">
                         <rect>
                              <x>20</x>
                              <y>20</y>
                              <width>381</width>
                              <height>241</height>
                         </rect>
                    </property>
                    <property name="styleSheet">
                         <string notr="true">QPushButton {
	color: rgba(255, 255, 255,210);
	background-color: rgba(120, 120, 130,199);
	border-color: rgba(120,120,130,199);
}
QPushButton:hover{
	background-color: rgba(90, 90, 100,220);
	border-color: rgba(150,150,160,220);
}
QPushButton:pressed {
	color: rgba(255, 255, 255,240);
	background-color: rgba(150, 150, 160,199);
	border-color: rgba(150,150,160,199);
}
QPushButton#btn_github{
	border-radius:5px;
	color: rgba(255, 255, 255,130);
	background-color: rgba(0,0,0,0);
}
QPushButton#btn_github:hover{
	border-radius:5px;
	color: rgba(225, 225, 225,130);
	background-color: rgba(0,0,0,0);
}
QPushButton#btn_github:pressed{
	border-radius:5px;
	color: rgba(255, 255, 255,170);
	background-color: rgba(0,0,0,0);
}
QPushButton#btn_logo{
	border-radius:5px;
	color: rgba(255, 255, 255,130);
	background-color: rgba(0,0,0,0);
}
QPushButton#btn_logo:hover{
	border-radius:5px;
	color: rgba(225, 225, 225,130);
	background-color: rgba(0,0,0,0);
}
QPushButton#btn_logo:pressed{
	border-radius:5px;
	color: rgba(255, 255, 255,170);
	background-color: rgba(0,0,0,0);
}
QPushButton#btn_close{
	background-color: rgba(0, 0, 0,0);
}
QPushButton#btn_close:hover{
	background-color: rgba(170, 70, 77,220);
}
QPushButton#btn_close:pressed{
	background-color: rgba(190, 90, 97,220);
}
QPushButton#btn_minimized{
	background-color: rgba(0, 0, 0,0);
}
QPushButton#btn_minimized:hover{
	background-color: rgba(90, 90, 100,220);
	border-color: rgba(150,150,160,220);
}
QPushButton#btn_minimized:pressed {
	color: rgba(255, 255, 255,240);
	background-color: rgba(150, 150, 160,199);
	border-color: rgba(150,150,160,199);
}</string>
                    </property>
                    <widget class="QPushButton" name="btn_github">
                         <property name="geometry">
                              <rect>
                                   <x>250</x>
                                   <y>157</y>
                                   <width>51</width>
                                   <height>51</height>
                              </rect>
                         </property>
                         <property name="font">
                              <font>
                                   <family>Social Media Circled</family>
                                   <pointsize>22</pointsize>
                              </font>
                         </property>
                         <property name="styleSheet">
                              <string notr="true">background-color: 0,0,0,0;</string>
                         </property>
                         <property name="text">
                              <string>)</string>
                         </property>
                    </widget>
                    <widget class="QLabel" name="label">
                         <property name="geometry">
                              <rect>
                                   <x>10</x>
                                   <y>40</y>
                                   <width>361</width>
                                   <height>191</height>
                              </rect>
                         </property>
                         <property name="styleSheet">
                              <string notr="true">background-color: rgba(40, 40, 47,220);
border-radius: 15px;</string>
                         </property>
                         <property name="text">
                              <string/>
                         </property>
                    </widget>
                    <widget class="QLineEdit" name="lineEdit">
                         <property name="enabled">
                              <bool>false</bool>
                         </property>
                         <property name="geometry">
                              <rect>
                                   <x>30</x>
                                   <y>100</y>
                                   <width>270</width>
                                   <height>22</height>
                              </rect>
                         </property>
                         <property name="font">
                              <font>
                                   <family>Roboto</family>
                                   <weight>75</weight>
                                   <bold>true</bold>
                              </font>
                         </property>
                         <property name="styleSheet">
                              <string notr="true">color: #ffffff;
background-color: rgba(120, 120, 130,199);
border-color: rgba(120,120,130,199);
border-radius:4px;</string>
                         </property>
                         <property name="text">
                              <string/>
                         </property>
                    </widget>
                    <widget class="QPushButton" name="btn_explorer">
                         <property name="geometry">
                              <rect>
                                   <x>310</x>
                                   <y>100</y>
                                   <width>41</width>
                                   <height>22</height>
                              </rect>
                         </property>
                         <property name="font">
                              <font>
                                   <pointsize>8</pointsize>
                                   <weight>50</weight>
                                   <bold>false</bold>
                              </font>
                         </property>
                         <property name="styleSheet">
                              <string notr="true">color: #ffffff;
border-radius:4px;</string>
                         </property>
                         <property name="text">
                              <string>°°°</string>
                         </property>
                         <property name="default">
                              <bool>true</bool>
                         </property>
                    </widget>
                    <widget class="QPushButton" name="btn_start">
                         <property name="enabled">
                              <bool>true</bool>
                         </property>
                         <property name="geometry">
                              <rect>
                                   <x>30</x>
                                   <y>155</y>
                                   <width>151</width>
                                   <height>51</height>
                              </rect>
                         </property>
                         <property name="font">
                              <font>
                                   <family>Roboto</family>
                                   <pointsize>16</pointsize>
                                   <weight>50</weight>
                                   <bold>false</bold>
                                   <underline>false</underline>
                                   <strikeout>false</strikeout>
                              </font>
                         </property>
                         <property name="styleSheet">
                              <string notr="true">border-radius: 10px;
                              </string>
                         </property>
                         <property name="text">
                              <string>I n i c i a r</string>
                         </property>
                    </widget>
                    <widget class="QLabel" name="label_2">
                         <property name="geometry">
                              <rect>
                                   <x>40</x>
                                   <y>160</y>
                                   <width>151</width>
                                   <height>51</height>
                              </rect>
                         </property>
                         <property name="styleSheet">
                              <string notr="true">border-radius: 10px;
color: rgba(255, 255, 255,210);
background-color: rgba(60, 60, 70,120);</string>
                         </property>
                         <property name="text">
                              <string/>
                         </property>
                    </widget>
                    <widget class="QLabel" name="label_3">
                         <property name="geometry">
                              <rect>
                                   <x>34</x>
                                   <y>108</y>
                                   <width>271</width>
                                   <height>21</height>
                              </rect>
                         </property>
                         <property name="styleSheet">
                              <string notr="true">border-radius:4px;
color: rgba(255, 255, 255,210);
background-color: rgba(60, 60, 70,120);</string>
                         </property>
                         <property name="text">
                              <string/>
                         </property>
                    </widget>
                    <widget class="QLabel" name="label_4">
                         <property name="geometry">
                              <rect>
                                   <x>315</x>
                                   <y>106</y>
                                   <width>41</width>
                                   <height>22</height>
                              </rect>
                         </property>
                         <property name="styleSheet">
                              <string notr="true">border-radius:4px;
color: rgba(255, 255, 255,210);
background-color: rgba(60, 60, 70,120);</string>
                         </property>
                         <property name="text">
                              <string/>
                         </property>
                    </widget>
                    <widget class="QLabel" name="label_5">
                         <property name="geometry">
                              <rect>
                                   <x>260</x>
                                   <y>167</y>
                                   <width>41</width>
                                   <height>41</height>
                              </rect>
                         </property>
                         <property name="font">
                              <font>
                                   <family>Social Media Circled</family>
                                   <pointsize>22</pointsize>
                              </font>
                         </property>
                         <property name="styleSheet">
                              <string notr="true">color: rgba(60, 60, 70,255);</string>
                         </property>
                         <property name="text">
                              <string>)</string>
                         </property>
                    </widget>
                    <widget class="QPushButton" name="btn_logo">
                         <property name="geometry">
                              <rect>
                                   <x>304</x>
                                   <y>157</y>
                                   <width>51</width>
                                   <height>51</height>
                              </rect>
                         </property>
                         <property name="font">
                              <font>
                                   <family>Untitled1</family>
                                   <pointsize>28</pointsize>
                              </font>
                         </property>
                         <property name="styleSheet">
                              <string notr="true"/>
                         </property>
                         <property name="text">
                              <string>A</string>
                         </property>
                    </widget>
                    <widget class="QLabel" name="label_7">
                         <property name="geometry">
                              <rect>
                                   <x>309</x>
                                   <y>162</y>
                                   <width>51</width>
                                   <height>51</height>
                              </rect>
                         </property>
                         <property name="font">
                              <font>
                                   <family>Untitled1</family>
                                   <pointsize>28</pointsize>
                              </font>
                         </property>
                         <property name="styleSheet">
                              <string notr="true">color: rgba(60, 60, 70,255);
background-color: rgba(0,0,0,0);</string>
                         </property>
                         <property name="text">
                              <string>A</string>
                         </property>
                    </widget>
                    <widget class="QLabel" name="label_8">
                         <property name="geometry">
                              <rect>
                                   <x>20</x>
                                   <y>75</y>
                                   <width>341</width>
                                   <height>60</height>
                              </rect>
                         </property>
                         <property name="styleSheet">
                              <string notr="true">background-color: rgba(255,255,255,20);
border-radius: 10px</string>
                         </property>
                         <property name="text">
                              <string/>
                         </property>
                    </widget>
                    <widget class="QLabel" name="label_9">
                         <property name="geometry">
                              <rect>
                                   <x>35</x>
                                   <y>80</y>
                                   <width>251</width>
                                   <height>16</height>
                              </rect>
                         </property>
                         <property name="font">
                              <font>
                                   <family>Roboto Black</family>
                                   <weight>75</weight>
                                   <bold>true</bold>
                              </font>
                         </property>
                         <property name="styleSheet">
                              <string notr="true">color: rgba(255,255,255,180);</string>
                         </property>
                         <property name="text">
                              <string>Elija la direccion del programa a bloquear</string>
                         </property>
                    </widget>
                    <widget class="QPushButton" name="btn_close">
                         <property name="geometry">
                              <rect>
                                   <x>319</x>
                                   <y>44</y>
                                   <width>41</width>
                                   <height>26</height>
                              </rect>
                         </property>
                         <property name="font">
                              <font>
                                   <family>Roboto Black</family>
                                   <pointsize>12</pointsize>
                                   <weight>75</weight>
                                   <bold>true</bold>
                              </font>
                         </property>
                         <property name="styleSheet">
                              <string notr="true">border-radius: 7px;</string>
                         </property>
                         <property name="text">
                              <string>X</string>
                         </property>
                    </widget>
                    <widget class="QPushButton" name="btn_minimized">
                         <property name="geometry">
                              <rect>
                                   <x>270</x>
                                   <y>44</y>
                                   <width>41</width>
                                   <height>26</height>
                              </rect>
                         </property>
                         <property name="font">
                              <font>
                                   <family>Roboto Black</family>
                                   <pointsize>12</pointsize>
                                   <weight>75</weight>
                                   <bold>true</bold>
                              </font>
                         </property>
                         <property name="styleSheet">
                              <string notr="true">border-radius: 7px;</string>
                         </property>
                         <property name="text">
                              <string>_</string>
                         </property>
                    </widget>
                    <zorder>label</zorder>
                    <zorder>label_8</zorder>
                    <zorder>label_7</zorder>
                    <zorder>label_2</zorder>
                    <zorder>btn_start</zorder>
                    <zorder>label_3</zorder>
                    <zorder>lineEdit</zorder>
                    <zorder>label_4</zorder>
                    <zorder>btn_explorer</zorder>
                    <zorder>label_5</zorder>
                    <zorder>btn_github</zorder>
                    <zorder>btn_logo</zorder>
                    <zorder>label_9</zorder>
                    <zorder>btn_close</zorder>
                    <zorder>btn_minimized</zorder>
               </widget>
          </widget>
          <widget class="QStatusBar" name="statusbar"/>
     </widget>
     <resources>
          <include location="../Images/QRC/res.py"/>
     </resources>
     <connections/>
</ui>
